docs/wallets.md

```diff
@@ -0,0 +1,41 @@
+---
+id: wallets
+title: Polkadot Wallets
+sidebar_label: Polkadot Wallets
+---
+
+## Polkadot Wallets
+
+The integration of a [wallet](https://wiki.polkadot.network/docs/en/glossary#wallet) and Polkadot
+allows for simple and easy access to private keys and signing transactions. Below are wallets
+supported by Polkadot.
+
```

Comment:
We should mention polkadot.js.org, Parity Signer, and polkadot.js plugin here and have them at the top.